Firmware/Apps/Src/main.c

```diff
@@ -0,0 +1,27 @@
+#include "stm32xx_hal.h"
+#include "Amperes.h"
+
+uint32_t reading;
+
+int main() {
+    HAL_Init();
+    SystemClock_Config();
+
+    // Initializing
+    if (Amperes_Init() != INIT_OK) return 0;
```

Comment:
call `error_handler` here so we know an error occurred
```suggestion
    if (Amperes_Init() != INIT_OK) error_handler();
```